tools/emptyDropsWrapper/test/prepRDS.R

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env Rscript
+library(nbHelpers)
+d <- read10xMatrix('raw_gene_bc_matrices/GRCh38', version='V2')
```

Comment:
I'm not familiar with R, but is `'raw_gene_bc_matrices/GRCh38'` something you would want to store in a variable so that this could be called on other input matrices?

Reply:
Fixed as part of addressing parametrization of the tests (below).